packaging/rpm/microshift-airgap.spec

```diff
@@ -0,0 +1,31 @@
+Name: microshift
+Version: %{version}
+Release: %{release}%{dist}
+# this can be %{timestamp}.git%{short_hash} later for continous main builds
+Summary: Create custom container storage
+License: ASL 2.0
+URL: https://github.com/redhat-et/microshift
+
+%description
+This rpm creates a custom RO container storage and pull images for disconnected scenarios.
+
+%prep
+
+%build
+
+
+%install
+mkdir -p /var/lib/microshift/airgap-images
```

Comment:
```suggestion
mkdir -p %{buildroot}/var/lib/microshift/airgap-images
```

Reply:
After that you can call Podman, I wonder if we could include you ssh as a source, but we need to use --root %{buildroot}/var/lib/microshift/container-images